tests/e2e/utils/pageObjects/wp/wpInitiativesInstructions.page.ts

```diff
@@ -0,0 +1,25 @@
+import { Locator, Page } from "@playwright/test";
+import BasePage from "../base.page";
+
+export class WPInitiativesInstructionsPage extends BasePage {
+  public path = "/wp/state-or-territory-specific-initiatives/instructions";
+
+  readonly page: Page;
+  readonly title: Locator;
+
+  constructor(page: Page) {
+    super(page);
+    this.page = page;
+    this.title = page.getByRole("heading", {
+      name: "State or Territory-Specific Initiatives Instructions",
+    });
+  }
+
+  public async fillFormFields() {
+    const radioButtons = await this.page.getByLabel("No").all();
+
+    for (const radio of radioButtons) {
+      await radio.click();
+    }
```

Comment:
This is an instance where, without the page in front of me, it's hard for me to guess what is happening in the app. What questions are being answered "No" to? How many of them are there?

This isn't necessarily a bad thing. For one, almost everyone reading this test ever _will_ have the app in front of them, or at least be familiar enough to remember this page. And I only bring it up because it contrasts to so much of the rest of the code - like the easily-parseable `this.fundingSources.getByRole("button", { name: "Edit" }).click()`, or the supremely readable `fundingPage.addSourceButton.click()`

Reply:
I do think this could be named better in the long run, but this is the "special" initiatives page that has only 2 radio groups:

<img width="814" alt="Screenshot 2024-09-30 at 3 35 57 PM" src="https://github.com/user-attachments/assets/88b2c9ba-2017-4a22-ad25-148cbd18c22d">

And in the interest of keeping these e2e tests as simple as possible, I've opted for the No option because I don't particularly care about the intricacies of making these two populations required for the initiative information in the following pages. Will make a note of renaming the variable to make this more clear!